examples/redefine-default-mounting/states/base/profile/GET.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (req, res, next) {
+  next();
```

Comment:
Is this a good example? Just sending to next without any condition? Maybe we can check a request header and decide will we handle this request or pass to the next. Because this code looks like very fragile.